cpp/src/string/nvcategory_util.cpp

```diff
@@ -1,13 +1,10 @@
 
 #include "nvcategory_util.hpp"
-
-
+#include <types.hpp>
 #include <nvstrings/NVCategory.h>
 #include <nvstrings/NVStrings.h>
 #include "rmm/rmm.h"
```

Comment:
While you are at it... 

```suggestion
#include <rmm/rmm.h>
```